endpoints-md.py

```diff
@@ -104,6 +104,8 @@ def from_type_specifier(typespec: TypeSpecifier) -> CompositeTypeLink:
             if payloadtype := typespec.payload_type:
                 if payloadtype.__module__ == "builtins":
                     return CompositeTypeLink.text_only(payloadtype.__name__)
+                if issubclass(payloadtype, Generic):  # type: ignore
```

Comment:
Can we remove that? I can't see the use case.
It also breaks previously working markdown links to `MigrationFile`, `SoftwarePackageUpdatePayload`...